jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.vue

```diff
@@ -27,11 +27,26 @@
       :items="dataset_items"
       :selected.sync="dataset_selected"
       :multiselect="multiselect"
-      :show_if_single_entry="false"
+      :show_if_single_entry="true"
       label="Data"
       hint="Select the data for photometry."
     />
 
+    <div v-if='config == "cubeviz"'>
+      <v-row class="row-no-outside-padding row-min-bottom-padding">
+        <v-col>
+          <v-text-field
+            v-model.number="cube_slice"
+            :readonly="true"
+            class="mt-0 pt-0"
+            type="number"
+            label="Slice (for cube)"
+            hint="Slice number (for cube only)"

```

Comment:
Since its read-only, let's disable to avoid confusion?  And maybe update the hint to explain a bit (feel free to change the text).

```suggestion
            hint="Extracts photometry from currently selected cube slice"
            disabled

```